pkg/services/ssosettings/ssosettingsimpl/service.go

```diff
@@ -365,9 +365,15 @@ func (s *Service) doReload(ctx context.Context) {
 	}
 
 	for provider, connector := range s.reloadables {
-		setting := getSettingByProvider(provider, settingsList)
+		settings := getSettingByProvider(provider, settingsList)
 
-		err = connector.Reload(ctx, *setting)
+		if settings == nil || len(settings.Settings) == 0 {
+			s.metrics.reloadFailures.WithLabelValues(provider).Inc()
```

Comment:
Please remove/comment out this line temporarily. We'll add it back in a follow up PR.

Reply:
Removed as requested.